docs/tutorials/general.md

```diff
@@ -0,0 +1,350 @@
+npm start
+```
+
+Next, access the API's OAS at [http://127.0.0.1:3000/openapi.json](http://127.0.0.1:3000/openapi.json) and save it to disk.
+
+Then change the server url to match the following. Otherwise, the GraphQL interface will not be able to make calls to the Example Family Tree API.
+
+```
+"servers": [
+  {
+    "url": "http://localhost:3000"
+  }
+]
+```
+
+### Install OASGraph
+
+To install OASGraph, clone the repository and link the library (for the CLI commands to work) using the indicated steps.
+
+```
+git clone git@github.com:strongloop/oasgraph.git
+cd oasgraph
+npm link
+```
+
+Please note that OASGraph can be used either as a library, or via its Command Line Interface (CLI). In this case, we will be using the CLI tool, which will start a server in addition to creating the GraphQL interface.
+
+### Start GraphQL server
+
+Start the GraphQL server by running the following command.
+
+```
+oasgraph [path to saved OAS]
+```
+
+The created GraphQL server is then accessible at [http://127.0.0.1:3001/graphql](http://127.0.0.1:3001/graphql).
+
+### Try simple queries
+
+Congratulations! You have created your first GraphQL interface with OASGraph.
+
+We can explore the GraphQL interface through the [GraphiQL IDE](https://github.com/graphql/graphiql) that will show at the previously provided link.
+
+Try to run a simple query like the following:
+
+GraphQL query
+```
+query{
+  person(id: 1){
+    name
+  }
+}
+```
+
+Expected output
+```
+{
+  "data": {
+    "person": {
+      "name": "Albert"
+    }
+  }
+}
+```
+
+***
+
+Although this is a fully-fledged GraphQL interface, we can still improve it.
+
+For instance, this GraphQL interface does not provide nested queries. Recall that the **Person object** also stores the *motherId* and *fatherId* which can be used to query other **People**. In GraphQL, we can make it so that a user can query the mother and father objects from within the child object. To do so, we will need to make slight additions to the OAS, which will be discussed in the following sections.
+
+For now, in the *Documentation Explorer* on the right hand side, click **Query** and then the return type, **Person**, in ```person(id: Float!): person```. Note that the **Person** object can only return the following fields. With our nested query changes, we will see new fields that will let us query the mother and father objects.
+
+```
+id: Float
+name: String
+generation: Float
+motherId: Float
+fatherId: Float
+```
+
+### Adding links
+
+[Link objects](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#linkObject "Link object documentation") are constructs in OAS that allow you to explicitly define relationships between different models and operations. For example, when you query for a **Person**, you will receive a *motherId* and a *fatherId*, which can be used to query for other **People**. When you have actions that naturally lead to the next, you use **Link objects** to describe this flow.
```

Comment:
It seems that the URL in this markdown link is not correct?!

Reply:
It works fine for me... Is there a problem?

Reply:
The URL is stated to be `https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#linkObject "Link object documentation"`. Is that really correct? Does the hash work for you?

Reply:
It works for me... I'm not sure what's wrong with it... Are you confused about the quotes at the end? That's not part of the link. That's just the text to shows up when you hover over the hyperlink.

Reply:
It should be `https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#linkObject`, no?

Reply:
It could be that... They're basically the same... The string at the end is part of the markdown language, just some text to display when your cursor is over the hyperlink. I could change it if you feel strongly about it.